### ISSUE TITLE
Feature Request : Add a method to get all groupIDs

### ISSUE THREAD

**Issue by whyorean** [tonyofrancis/Fetch#413]
Hey there, thanks for the awesome library.

`Do we have a method to get a list of all queued/active groupIDs ?`

If not, would you please consider adding it ? If its not too much of a work.
Also if this can be achieved with current codebase, kindly let me know about the workarounds.

Thanks

**Comment by tonyofrancis** [comment]
@whyorean  This issue is now fixed in new Fetch version 3.0.10/Androidx 3.1.4.
```java
        fetch.getAllGroupIds(new Func<List<Integer>>() {
            @Override
            public void call(@NotNull List<Integer> result) {
                //get all group ids from list.
            }
        });
```

**Comment by whyorean** [comment]
Perfect ! Thanks
